tests/console/container_base_images.pm

```diff
@@ -38,6 +38,16 @@ sub run_image_tests {
     }
 }
 
+sub upload_image_logs {
+    # Rename for better visibility in Uploaded Logs
+    if (script_run('mv /var/tmp/container_base_images_log.txt logs.txt') != 0) {
```

Comment:
I personally prefer keeping the name `container_base_images_log.txt`, cause it will help identifying where the logs come from. `logs.txt` is too generic and could come from any module in the job.

Reply:
Changed.

Reply:
Having the filename `logs.txt` is fine as it results in the logfile `container_base_images-logs.txt` (See http://pdostal-server.suse.cz/tests/8918#downloads)
Changing the filename to `container_base_images-logs.txt` results in a filename called `container_base_images-container_base_images-logs.txt`

Reply:
Yeah, you are right. @pdostal: probably you can change it back as it was before my comment? Sorry for the trouble!

Reply:
No worries :-)